servicediscovery/requests.go

```diff
@@ -0,0 +1,12 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package servicediscovery
+
+type ResolveRequest struct {
+	ID       string
+	GrpcPort int
```

Comment:
I think this could be any port :) wouldn't it be good to rename to `port`?

Reply:
I agree, rename to `Port`.

Reply:
Yep